static/themes/default-theme.scss

```diff
@@ -331,4 +331,10 @@ div.argmenuitem span.argdescription {
 
 .ts-dropdown .optgroup-header {
     background-color: #f6f6f6;
- }
+}
+
+.currentCursorPosition {
+    color: #15a3b9;
```

Comment:
Contrast is too low in the light theme, this might be better being a darker gray / outright black

Reply:
Do you think it's ok to merge though Ruben? You're our colour expert :-)

Reply:
Yes! By all means